### ISSUE TITLE
Attaching a manyToMany relation with translatable pivot field does not work

### ISSUE THREAD

**Issue by v1nk0** [mrmonat/nova-translatable#16]
Simplified example:

Resource: ``Post`` (db-table ``posts``)
Resource: ``Attribute`` (db-table ``attributes``)

There is a belongsToMany relation between the two (db-table ``attribute_post``). 
This relation has one pivot field called ``value``.
It's a translatable field. 
There is a model (`PostAttributes`) for the relation as well:

**App\Post.php** (attribute relation)
```php
    public function attributes()
    {
        return $this->belongsToMany( Attribute::class )
            ->withPivot(['value'])
            ->using( PostAttributes::class );
    }
```

**App\Attribute.php** (post relation)
```php
    public function posts()
    {
        return $this->belongsToMany( Post::class )
            ->withPivot('value')
            ->using( PostAttributes::class );
    }
```

**App\PostAttributes.php**
```php
use Spatie\Translatable\HasTranslations;
use Illuminate\Database\Eloquent\Relations\Pivot;

class PostAttributes extends Pivot
{
    use HasTranslations;
    public $translatable = ['value'];
}

```

Whenever the relation exists in the database it can be edited using Nova.
But when I want to attach a new ``Attribute`` to the ``Post`` I get the following error:
<img width="669" alt="screen shot 2018-09-14 at 15 10 58" src="https://user-images.githubusercontent.com/8701964/45552187-69eed180-b830-11e8-85c5-9ff450d55196.png">

**Comment by mrmonat** [comment]
Can you please share your "fields" Method in your Nova Resource?

**Comment by v1nk0** [comment]
Yes, that would be:

```php
BelongsToMany::make('Attribute', 'attributes', 'App\Nova\Attribute')
    ->fields(function () {
        return [
            Translatable::make('Wert', 'value')
                ->indexLocale('de')
                ->singleLine()
        ];
    }),
```

**Comment by mrmonat** [comment]
Fixed in Version 1.1.3 (https://github.com/mrmonat/nova-translatable/releases/tag/1.1.3)
